lib/fog/compute/google/models/network.rb

```diff
@@ -8,19 +8,24 @@ class Google
       class Network < Fog::Model
         identity :name
 
-        attribute :kind
-        attribute :id
-        attribute :ipv4_range, :aliases => "IPv4Range"
-        attribute :creation_timestamp, :aliases => "creationTimestamp"
+        attribute :auto_create_subnetworks, aliases => "autoCreateSubnetworks"
+        attribute :creation_timestamp, aliases => "creationTimestamp"
         attribute :description
-        attribute :gateway_ipv4, :aliases => "gatewayIPv4"
-        attribute :self_link, :aliases => "selfLink"
+        attribute :gateway_i_pv4, aliases => "gatewayIPv4"
```

Comment:
Anyway we can keep `ip` together? Or is this Rubocop complaining?

Reply:
I changed it to match the argument in the Google Client API. I'm not sure if we want it to vary - i could add it as an alias? Mostly we would have to do some conversion during insert.

Reply:
Ah ok. Let us do an alias. If it's an alias, I think the conversion should happen automatically.

Reply:
done!